pylot/component_creator.py

```diff
@@ -544,10 +552,10 @@ def add_planning(goal_location, pose_stream, prediction_stream,
     return waypoints_stream
 
 
-def add_control(pose_stream,
-                waypoints_stream,
-                ground_vehicle_id_stream=None,
-                perfect_obstacles_stream=None):
+def add_control(waypoints_stream,
+                ground_vehicle_id_stream: Stream[int],
+                perfect_obstacles_stream: Stream[obstacle],
+                pose_stream: Optional[Stream[pylot.utils.Pose]] = None) -> Tuple[Stream, Stream]:
```

Comment:
Please make sure the arguments stay in the same order. Also, the return type is only a single stream.